File: push/config.sample.php
<?php

$C["source"] = [
	"gadgetlocal" => "/home/user/sample/",
	"gadgetgithub" => "https://raw.githubusercontent.com/user/repo/master/"
];

$C["target"] = [
	"sample" => "User:Example/sample/",
	"gadget" => "User:Example/gadget/"
];

$C["web"] = [
	"beta" => [
		"wikiapi" => "https://zh.wikipedia.beta.wmflabs.org/w/api.php",
		"user" => "Example",
		"pass" => "",
		"bot" => false,
		"minor" => true,
		"nocreate" => true,
		"cookiefile" =>__DIR__."/../tmp/push-cookie.txt"
	],
	"zhwp" => [
		"wikiapi" => "https://zh.wikipedia.org/w/api.php",
		"user" => "Example",
		"pass" => "",
		"bot" => false,
		"minor" => true,
		"nocreate" => true,
		"cookiefile" => __DIR__."/../tmp/push-wp-cookie.txt"
	]
];

$C["project"] = [
	"sample" => [
		"source" => [
			"gadgetlocal",
			"gadgetgithub",
		],
		"target" => [
			"sample",
			"gadget"
		],
		"web" => [
			"beta",
			"zhwp"
		],
		"summary" => "deploy new feature",
		"files" => [
			'sample.js' => 'sample.js'
		]
	]
];

$C["User-Agent"] = "";

File: push/python/config.sample.py
# -*- coding: utf-8 -*-
import os

BASEDIR = os.path.dirname(os.path.abspath(__file__))

cfg = {}

cfg['source'] = {
    'gadgetlocal': '/home/user/sample/',
    'gadgetgithub': 'https://raw.githubusercontent.com/user/repo/master/',
}

cfg['target'] = {
    'sample': 'User:Example/sample/',
    'gadget': 'User:Example/gadget/',
}

cfg['web'] = {
    'meta': {
        'bot': False,
        'minor': False,
        'nocreate': False,
    },
    'zhwp': {
        'bot': False,
        'minor': False,
        'nocreate': False,
    },
}

cfg['project'] = {
    'sample': {
        'source': [
            'gadgetlocal',
            'gadgetgithub',
        ],
        'target': [
            'sample',
            'gadget',
        ],
        'web': [
            'meta',
            'zhwp',
        ],
        'summary': 'deploy new feature',
        'files': {
            'sample.js': 'sample.js',
        }
    },
}
